documentation/API-Readiness-Checklist.md

```diff
@@ -1,11 +1,13 @@
 # API Readiness minimum criteria checklist
-| No | Deliverables/Criteria      | Mandatory | Reference template                          |
+**PLEASE NOTE: This document is deprecated as the updated [API Readiness Checklist](https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md) was developed within Release Management Working Group.**
+
+~~| No | Deliverables/Criteria      | Mandatory | Reference template                          |
```

Comment:
Instead of marking as ~~older~~

I would add something like the suggestion (or removed completely the table and in RM PR add a note for User Stories template